### DISCUSSION TITLE
EASYIQ ugeplan

### DISCUSSION THREAD

**Discussion by KaspersArk** [scaarup/aula#81] [Ideas]
Hej Scaarup!

Har været bruger af din lækre intregration længe og har lavet en lækkert overblik, uden for AULA helvede i mit HA setup ;)
Vores skole (måske andre skoler også) er overgået fra ugeplanen i MeeBook til ugeplaner i EASYIQ.
Er der nogen mulighed for og få dette som en del af integrationen også? 🥇 

https://easyiq.dk/aula/
https://easyiq.dk/easyiq-ugeplans-widget/

**Comment by scaarup** [comment]
Heysa @KaspersArk 

Fedt det kan bruges 👍🏻.

Jo det skulle da nok være muligt. Hvordan ser de her ugeplaner ud - bliver de framet inde i Aula eller bliver du sendt videre fra Aula til EasyIQ? Skal du logge ind der, eller er det nok blot at logge ind i Aula?

**Reply by KaspersArk** [reply]
Mine knægte bruger det dagligt til skema og ugeplener, dog uden ugeplaner da de jo ligger i EasyIQ...
De er framed i AULA og kræver kun login i Aula og har derved adgang ligesom ugeplanerne fra meebook.
Man skal dog ind i kalender og sidste faneblad i appen hedder nu EasyIQ..
Vedhæfter lige et screenshot 😉
På forhånd tak...
Kasper

![Screenshot_20230205-155943](https://user-images.githubusercontent.com/76815973/216828587-f2c4f940-3dbd-4419-a99a-d80961cb9d27.jpg)
